test/models/pawn_test.rb

```diff
@@ -1,4 +1,102 @@
 require 'test_helper'
 
 class PawnTest < ActiveSupport::TestCase
+  def setup
+    @game = Game.new
+    @game.populate_board!
+    @first_white_pawn = Pawn.create(game: @game, color: 'white', x_position: 3, y_position: 6)
```

Comment:
if you're creating pawns on your own, it's probably a good idea to not run the `populate_board!` method, because now you'll have dups pieces on top of each other.

[CANNON STACK](https://youtu.be/yt-yjvbnAUQ?t=16)